Dockerfile

```diff
@@ -1,12 +1,19 @@
 FROM python:3.8-slim
-RUN pip install --no-cache-dir pipenv
+
+RUN useradd -d /app -m appuser
+USER appuser
+
 WORKDIR /app
+
 COPY Pipfile* /app/
-RUN cd /app && \
-    pipenv install --system --deploy
+COPY *.py /app/
+COPY LICENSE /app/
```

Comment:
```suggestion
COPY *.py LICENSE /app/
```

Reply:
@joschi36 Do you want me to do the same COPY consolidation for all the other non-root PRs?